content/02.introduction.md

```diff
@@ -78,29 +74,49 @@ The symptoms of infection by SARS-CoV-2 can vary greatly, making it difficult fo
-Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States and Europe, it was available soon afterward as a stand-alone vaccine that was eventually incorporated into the commercially available seasonal influenza vaccines [@doi:10.1056/NEJMp2005630].
-If H1N1 vaccine development provides any indication, considering developing and manufacturing platforms for promising COVID-19 vaccine trials early could hasten the emergence of an effective prophylactic vaccine against SARS-CoV-2. 
+Understanding how the virus functions and interacts with the host is foundational to understanding pathogenesis and disease progression and to identifying available and novel approaches to treatment.
+Therefore, the fields of virology, immunology, and molecular biology are fundamental to characterizing SARS-CoV-2 and COVID-19.
+These topics can be approached using a range of techniques, including characterization of the host response from the cellular to systems level.
+Contextualizing SARS-CoV-2 in relation to other viruses that infect humans and other animals can further serve to elucidate the reaction of the human host to viral exposure.
+This information, when combined with an understanding of the biology of pharmaceutical and medical interventions, can guide new approaches to treatment.
 
-Unlike many global vaccine development programs previously, such as with H1N1, the vaccine development landscape for COVID-19 includes vaccines produced by a wide array of technologies. 
-Experience in the field of oncology is encouraging COVID-19 vaccine developers to use next-generation approaches to vaccine development, which have led to the great diversity of vaccine development programs [@url:https://www.the-scientist.com/news-opinion/newer-vaccine-technologies-deployed-to-develop-covid-19-shot-67152]. 
-Diverse technology platforms include DNA, RNA, virus-like particle, recombinant protein, both replicating and non-replicating viral vectors, live attenuated virus, and inactivated virus approaches. 
-Given the wide range of vaccines under development, it is possible that some vaccine products may eventually be shown to be more effective in certain subpopulations, such as children, pregnant women, immunocompromised patients, the elderly, etc.
+#### Public Health
 
-#### Diagnostics and Therapeutics for Viruses
+One necessary component of determining how to manage the outbreak is to understand epidemiological factors related to the transmission of the SARS-CoV-2 virus. 
+These can include characteristics such as when infected individuals are contagious, how the virus is transmitted between individuals, the range of symptoms associated with infection and/or contagiousness in different individuals, and how rapidly the virus propogates between individuals, etc.
+The development of diagnostic tools is critical to this goal.
```

Comment:
you mean better or more accurate diagnostic tools? because tools exist already

Reply:
I think we are aiming to review all of the diagnostic tools developed for SARS-CoV-2 (even if some of them were developed in January) so I think this works -- but let me know if I'm missing something!